test/acceptance/features/bindWithCustomNamingStrategies.feature

```diff
@@ -7,11 +7,29 @@ Feature: Bind an application to a service using custom naming strategies
     Background:
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
-        * PostgreSQL DB operator is installed
+        * The Custom Resource Definition is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backends.stable.example.com
+            spec:
+                group: stable.example.com
+                versions:
+                  - name: v1
+                    served: true
+                    storage: true
+                scope: Namespaced
+                names:
+                    plural: backends
+                    singular: backend
+                    kind: Backend
+                    shortNames:
+                      - bk
+            """
```

Comment:
Let's follow DRY and avoid copying - wrap this into a step like `CustomResourceDefinition backends.stable.example.com is available`

Reply:
PTAL.